src/components/videos/VideosList.test.js

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from '@testing-library/react';
+import VideosList from './VideosList';
+import mockData from '../../helper/mock-data.json';
+
+describe('VideosList Component', () => {
+  test('should not render nothing if props.list is empty', () => {
+    render(<VideosList list={[]} />);
+    expect(screen.queryByRole('listitem')).toBeFalsy();
```

Comment:
when testing render it is a better idea `.toBeInDocument` [Source](https://testing-library.com/docs/guide-disappearance/#nottobeinthedocument)

Reply:
Ok.